web/src/pages/Setting/set-security.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
```

Comment:
You should remove this, this directory is already pushed in current under a new name